### ISSUE TITLE
Feature Request: Improve Lighthouse Performance Score

### ISSUE THREAD

**Issue by wordshaker** [dotnetnotts/dotnetnotts-web#99]
### Describe the feature

The site currently has a score of 17 for performance when Lighthouse is run against it. 

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110366267-f5323200-803d-11eb-8f6e-5bcdb08ed7e6.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the accessibility of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse score for performance is increased to at least 90.
